atlp-rwanda/commanders-rn-medica
# Feature: book doctor appointments screens

#### What does this PR do?

This PR includes designs of booking an appointment with a doctor

#### Description of Task to be completed?

- Design doctor details screen
- Design ratings & reviews screen
- Design book appointment screen
- Design Packages screen
- Design Patients details screen 

#### How should this be manually tested?

Navigate to the home page and scroll down. Click on one doctor to view their details. At the bottom you can view the reviews of a doctor click on see all to  list all reviews. After that go back one step and click on `Book Appointment` button, a list of available hours and a calendar will show. Click on `Next` button to go to the page for selecting packages and finally click on `Next` Button to go to the patient details page

#### Any background context you want to provide?

The calendar component provides a prop for rendering custom header as its default is a little different from our design but it doesn't provide the proper function to change months while clicking on the icons. I changed the icons look but their position remains the same.

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/8d9e5f63-ed29-4dec-b473-2d341179d8a4" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/e81c16fb-9766-4f52-9265-fb9929943a68" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/4710d770-8574-42a0-b0aa-9b550ca134ef" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/0483d9fa-0e32-4d1b-89d3-c7674382ae8f" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/00507b21-a8ef-49bd-afac-60d5c2c1a787" width="150" />

#### Questions:

N/A
